services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/model.tsx

```diff
@@ -19,11 +19,22 @@ const Model = ({ match }) => {
     queryFn: fetchModel
   });
 
-  // function handleSetModelNotes(text: string){ setModelNotes(text) };
+  const initialNoteValue = !modelData ? '' : modelData.model.notes;
```

Comment:
We generally want to evaluate for truthiness first, so this should be `modelData ? modelData.model.notes : ''`